### ISSUE TITLE
Preset option is not taken into account

### ISSUE THREAD

**Issue by clemcode** [nuxt/image#261]
Hi, 

here are details about the `preset` option problem I faced : 

## What happens ?
The hero preset is found (no "unfound preset" error) but doesn't modify the image (image keeps its initial size)

## Reproduction

```html
<template>
  <nuxt-img preset="hero" src="/my-image.jpg" />
</template>
```
```javascript
/* nuxt.config.js */

export default {
  image: {
    presets: {
      hero: {
        modifiers: {
          fit: "cover",
          format: "jpg",
          width: 800,
          height: 300,
        },
      },
    },
  },
};
```

**Comment by pi0** [comment]
Thanks for issue. Should be fixed in next release.

**Comment by lucadalli** [comment]
@pi0  I am still facing this issue on `0.4.9`

**Comment by pi0** [comment]
#274 should possibily fix this as well (trying before close)

**Comment by pi0** [comment]
It should be finally fixed in 0.4.10+ (I could locally confirm). Please ping here if still `preset` is not working.

**Comment by clemcode** [comment]
Hi,

the `sizes` property doesn't seem to work in `presets`.

Nuxt version: 2.15
Nuxt Image version: 0.4.14
[CodeSandbox](https://codesandbox.io/s/spring-glitter-31c65?file=/pages/index.vue) (l.39 and below)

Output from `<nuxt-img/>` when a preset with `sizes` is applied:

`<img src="/_ipx/featured.jpeg?sizes=sm:90vw+md:80vw+lg:30vw" alt="Clothes displayed on bed">`
